fbw-a380x/src/systems/shared/AircraftVersionChecker.ts

```diff
@@ -0,0 +1,277 @@
+// Copyright (c) 2022 FlyByWire Simulations
```

Comment:
This file also exists in `fbw-common/` I think? It seems like that's also the case for a few other files in `shared/`.

Reply:
Ah yeah, it's not used anyways so I deleted it. I also cleaned up the shared folder a bit. There's some duplicate stuff in there, but that's best sorted out in another PR. autopilot.ts for example is needed, and not existing in fbw-common but in a32nx/shared and a380x/shared